source/introduction/history-of-ethereum.rst

```diff
@@ -1,3 +1,70 @@
 ********************************************************************************
 History of Ethereum
-********************************************************************************
\ No newline at end of file
+********************************************************************************
+
+Inception
+================================================================================
+Ethereum was initially described by Vitalik Buterin in late 2013 as a result of his research and work in the Bitcoin community. Shortly thereafter, Vitalik published the `Ethereum white paper <http://vbuterin.com/ethereum.html>`_, where he describes in detail the technical design and rationale for the Ethereum protocol and smart contracts architecture. In January 2014, Ethereum was formally announced by Vitalik at the The North American Bitcoin Conference in Miami, Florida, USA. From that moment on, the excitement around Ethereum and the its momentum has grown at a dramatic pace. 
+
+Around that time, Vitalik also started working with Dr. Gavin Wood and together co-founded Ethereum. By April 2014, Gavin published the `Ethereum Yellow Paper <http://gavwood.com/paper.pdf>`_ that would serve as the technical specification for the Ethereum Virtual Machine (EVM). By following the detailed specification in the Yellow Paper, the Ethereum client has been implemented in six programming languages (C++, Go, Python, Java, JavaScript, and Haskell), and has resulted in better software overall.
+
+.. https://blog.ethereum.org/2016/02/09/cut-and-try-building-a-dream/
+.. http://www.coindesk.com/ethererum-launches-cryptocurrency-2-0-network/
+
+
+The Ethereum Foundation and Crowdsale
+================================================================================
+In addition to developing the software for Ethereum, the ability to launch a new cryptocurrency and blockchain requires a massive bootstrapping effort in order to assemble the resources needed to get it up and running. To kickstart a large network of developers, miners, investors, and other stakeholders, Ethereum announced its plan to conduct a crowdsale of ether tokens, the currency unit of Ethereum. The legal and financial complexities of raising funds through a crowdsale led to the creation of several legal entities, including the Stiftung Ethereum (Ethereum Suisse), established in June 2014 in Zug, Switzerland. 
```

Comment:
Thanks. I like that term better. Crowdsale is somehow the prevalent term in the ethereum blog and other sources. I will change it